configure.cmake

```diff
@@ -556,6 +556,11 @@ IF(WITH_PERCONA_TELEMETRY)
   SET(HAVE_PERCONA_TELEMETRY 1)
 ENDIF()
 
+IF(WITH_PERCONA_PROBUILD OR PROBUILD)
+  SET(MYSQL_SERVER_SUFFIX "-pro")
+  SET(HAVE_PERCONA_PROBUILD 1)
```

Comment:
what is the difference between SET and ADD_DEFINITIONS?

Reply:
Probably you will not see `-DHAVE_PERCONA_PROBUILD` from command line because it will be defined in `config.h`.

Reply:
Ah, I see. I used ADD_DEFINITIONS and I see -DPROBUILD in command line.